<!DOCTYPE html>
<html>
<head>
  <title>MPAC CT</title>
  <style>
    body {
      font-family: 'Courier New', Courier, monospace, sans-serif;
      max-width: 600px;
      margin: auto;
      padding: 1em;
      line-height: 1.6;
    }
    h1 {
      color: #2c3e50;
    }
    
  </style>
</head>
<body>
  <h1>MPAC CT</h1>
  <p>
    This
    <a href="https://genome.ucsc.edu/cgi-bin/hgTracks">UCSC Genome Browser</a>
    track displays predicted variant effects  in cell-line CT for all variants in gnomAD v3.1.2 on genome build hg38. We predict alternate and reference allele MPRA (Massively Parallel Reporter Assay) activity and compute the allelic skew between them. We used a machine-learning model called MPAC (Malinois with Parallel Aggregated Cross-validation) to do this. This is part of
    <a href="https://www.biorxiv.org/content/10.1101/2025.04.16.648420v1">Identifying non-coding variant effects at scale via machine learning models of cis-regulatory reporter assays</a>.
    Read the paper for more details.
  </p>
  <p>
    <a href="https://github.com/Reilly-Lab-Yale/MPAC_gnomAD_and_satmut">Relevant code can be found on our github.</a> 
    For code specifically used to generate these tracks, consult `MPAC_gnomAD_preprocessing/helper/ucsc` within the repository.
  </p>
  <p>
    Hosting kindly provided by ....
  </p>


  <h2>Contact</h2>
  <p>This track was generated by <a href="mailto:mackenzie.noon@yale.edu">Mackenzie Noon</a></p>
  <p>For questions, contact the corresponding authors of
    <a href="https://www.biorxiv.org/content/10.1101/2025.04.16.648420v1">Identifying non-coding variant effects at scale via machine learning models of cis-regulatory reporter assays</a>.
  </p>

  <h2>Preview</h2>
  <img src="https://vcf-mpac-test.s3.us-east-1.amazonaws.com/ucsc_demo.png" alt="Track Preview" width="600" />
  <p>
    Red dots are emvars, blue dots are non-emvars. Y axis is skew. Tooltips show the alternate allele. 
  </pp>
</body>
</html>